src/CMakeLists.txt

```diff
@@ -264,6 +264,10 @@ endif()
 # Disable BOOST autolink
 add_definitions(-DBOOST_ALL_NO_LIB)
 
+#To be removed later, a bug to make things work with current vcpkg
+#https://github.com/microsoft/vcpkg/issues/22495
+add_definitions(-DBOOST_USE_WINAPI_VERSION=BOOST_WINAPI_VERSION_WIN7)
```

Comment:
maybe add it as optional with a cmake option, not everybody might need this if they are not using vcpkg